apps/02RockPaperScissors.js

```diff
@@ -9,15 +9,40 @@ var rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase();
+  hand2 = hand2.toLowerCase();
 
   // Write code here
+  if (hand1 === hand2) {
+    return "It's a tie!";
+  } else if (hand1 === "rock") {
 
+    if (hand2 === "scissors") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+  } else if (hand1 === "paper") {
+
+    if (hand2 === "rock") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+  } else if (hand1 === "scissors") {
+
+    if (hand2 === "rock") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+  }
```

Comment:
here for improvement there could be an add in code snippet for unrecognized entries. i attempted this but didnt succeed.